package.json

```diff
@@ -22,23 +22,24 @@
   },
   "homepage": "https://github.com/electron-userland/electron-download#readme",
   "dependencies": {
-    "debug": "^2.2.0",
+    "debug": "^3.0.0",
     "env-paths": "^1.0.0",
-    "fs-extra": "^2.0.0",
+    "fs-extra": "^4.0.1",
     "minimist": "^1.2.0",
-    "nugget": "^2.0.0",
+    "nugget": "^2.0.1",
     "path-exists": "^3.0.0",
-    "rc": "^1.1.2",
-    "semver": "^5.3.0",
-    "sumchecker": "^2.0.1"
+    "rc": "^1.2.1",
+    "semver": "^5.4.1",
+    "sumchecker": "^2.0.2"
```

Comment:
I still don't think it's worth bumping unless it's necessary for a new feature/bugfix.